src/ui/MobileSearch/Insights.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+  import Suggestions from './Suggestions.svelte'
+  import { queryInsights, queryInsightsBySearchTerm } from './api'
+
+  export let searchTerm
+  export let filter
+
+  let loading = true
+  let items = [] as any[]
+
+  $: searchTerm ? onInput(searchTerm) : getInsights()
+
+  function onInput(searchTerm: string) {
+    loading = true
+    queryInsightsBySearchTerm(searchTerm).then((data) => {
+      items = data
+      loading = false
+    })
+  }
+
+  function getInsights() {
+    loading = true
+    queryInsights().then((data) => {
+      items = data
+      loading = false
+    })
+  }
+</script>
+
+<Suggestions {items} {loading} {filter} let:item onItemClick={window.__onLinkClick}>
+  <div class="wrapper row hv-center">
+    <img src="{process.env.MEDIA_PATH}/icons/description-filled.svg" alt="Insights" class="icon" />
```

Comment:
Use `Svg`

Reply:
I used img tag here because SVG can not render it properly